locale/en_US/submission.xml

```diff
@@ -70,6 +71,7 @@
 	<message key="submission.submit.form.authorRequiredFields">The first name, last name, and email address of each author is required.</message>
 	<message key="submission.submit.form.titleRequired">Please enter the title of your article.</message>
 	<message key="submission.submit.form.abstractRequired">Please enter your article's abstract.</message>
+	<message key="submission.submit.form.wordCountAlert">You have exceeded the maximum word limit for abstracts in this section's track.</message>
```

Comment:
I think you can re-use the string from above. But either way, it should include the maximum word count in the error message.

Reply:
Here I am not able to pass the parameter to the keys/messages in the FormValidator, s. https://github.com/pkp/pkp-lib/blob/master/classes/form/validation/FormValidator.inc.php#L78.
So I can leave it as it is?

Reply:
Is it possible to use FormValidatorLength? https://github.com/pkp/pkp-lib/blob/master/classes/form/validation/FormValidatorLength.inc.php

Reply:
> Your abstract is too long. Please shorten it to below the word limit indicated for this section.